src/catalog/ddl.c

```diff
@@ -1399,6 +1404,29 @@ orioledb_utility_command(PlannedStmt *pstmt,
 	free_parsestate(pstate);
 }
 
+static void
+o_validate_replica_identity(Relation rel, ReplicaIdentityStmt *stmt)
+{
+	elog(LOG, "Current replident %c, setting replident %c", rel->rd_rel->relreplident, stmt->identity_type);
```

Comment:
Might be better to log this message in DEBUG level instead of LOG

```suggestion
	elog(DEBUG1, "Current replident %c, setting replident %c", rel->rd_rel->relreplident, stmt->identity_type);
```